### PR TITLE
initial pull request

### PR DESCRIPTION
**Pull request by hneth**
Only some **minor edits** (mostly of a cosmetic nature) — just to establish a workflow, prior to making more substantial changes.  If this works, I'd start editing the vignettes next...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,35 +134,37 @@ plot(heart.fft,
...
 ``` r
+
 # Compare results across algorithms in test data
 heart.fft$competition$test
 #>   algorithm   n hi fa mi cr      sens   spec    far       ppv       npv
 #> 1   fftrees 153 64 19  9 61 0.8767123 0.7625 0.2375 0.7710843 0.8714286
 #> 2        lr 153 55 13 18 67 0.7534247 0.8375 0.1625 0.8088235 0.7882353
 #> 3      cart 153 50 19 23 61 0.6849315 0.7625 0.2375 0.7246377 0.7261905
-#> 4        rf 153 58  8 15 72 0.7945205 0.9000 0.1000 0.8787879 0.8275862
+#> 4        rf 153 58 11 15 69 0.7945205 0.8625 0.1375 0.8405797 0.8214286
```
[the remark this answers is not in the document]
Comment (hneth) [reply README.md]: Yes, it seems that RF yields slightly different results every time.  I've added a `set.seed()` statement to enforce reproducible randomness.

**Comment by hneth** [comment]
Hope this works now.

**Comment by hneth** [comment]
Accidentally closed this pull request (with last comment) — thus reopening it now...

**Review by ndphillips — APPROVED** [review]

**Comment by ndphillips** [comment]
Looks good, thanks @hneth ! Feel free to merge

**Comment by hneth** [comment]
Thanks for approving these changes.  However, I don't see an option for merging — only a message "Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests."  Can you help?

**Comment by ndphillips** [comment]
ah got it. Just sent you an invite to be a collaborator (and merged the PR)

**Pull request merged**